movie.sh

```diff
@@ -24,45 +24,22 @@ do
 done < "$input"
 rm mov.txt
 rm submov.txt
-rate=$(printf %.2f $rating)
+rate=$(printf %.1f $rating)
 echo $rate
 }
 
-
+rm out.txt
 ls movies > movlist.txt
```

Comment:
This statement assumes that the `movies` directory will be in the same directory as I am executing the script it. It will be better if we can make this configurable via command line arguments so that we can execute the script from anywhere and point it to a directory with movie files anywhere on the filesystem.

Can you figure this out?